app/javascript/lib/efile_security_information.js

```diff
@@ -0,0 +1,22 @@
+import initFingerprint2 from "../vendor/IRS_fingerprint2_2_1_2";
+import initCryptoJS from "../vendor/sha1";
+
+export function getEfileSecurityInformation() {
+    let Fingerprint2 = initFingerprint2();
+    let CryptoJS = initCryptoJS();
+    Fingerprint2.get(function(components) {
+        var stringtohash = components.map(function (pair) { return pair.value }).join('###')
+        let encrypted_device_id =  CryptoJS.SHA1(stringtohash);
+        document.getElementById('ctc_consent_form_device_id').value = encrypted_device_id || "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
```

Comment:
i think it would be easier to see when we're not getting the encrypted_device_id back from the browser correctly by setting this default elsewhere. We can set the default on the model, _or_ we can allow this value to be blank and format to the default when creating the XmL